flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/PartitionRequestClientHandlerTest.java

```diff
@@ -169,95 +182,19 @@ public void testCancelBeforeActive() throws Exception {
 		client.cancelRequestFor(inputChannel.getInputChannelId());
 	}
 
-	/**
-	 * Tests that an unsuccessful message decode call for a staged message
-	 * does not leave the channel with auto read set to false.
-	 */
-	@Test
-	@SuppressWarnings("unchecked")
-	public void testAutoReadAfterUnsuccessfulStagedMessage() throws Exception {
```

Comment:
should we instead have a simple test that verifies that the `CreditBasedClientHandler` throws in case it has no buffers available? (which is a violation of the protocol but may happen for whatever reason)

Reply:
I will consider it